submissions/korotushak96/js-dom/index.css

```diff
@@ -0,0 +1,143 @@
+@media screen {
+    .burger {
+        position: absolute;
+        top: 17px;
+        right: 10px;
+        cursor: pointer;
+    }
+
+    .bar1, .bar2, .bar3 {
+        width: 35px;
+        height: 5px;
+        background-color: #333;
+        margin: 6px 0;
+        transition: 0.4s;
+    }
+
+    .change .bar1 {
+        transform: rotate(-45deg) translate(-9px, 6px) ;
+    }
+
+    
+    .change .bar2 {
+        opacity: 0;
+    }
+
+    
+    .change .bar3 {
+        transform: rotate(45deg) translate(-8px, -8px) ;
+    }
+
+    .container{
+        position: relative;
+    }
+
+    .menu{
+        position: absolute;
+        font-size: 30px;
+        color: blue;
+        transform: translate(-300px, 0px);
+        transition: 0.5s;
+    }
+
+    .menu.open{
+        transform: translate(0px, 0px);
+    }
+
+    .menu-item{
+        box-sizing: border-box;
```

Comment:
I noticed you are using `box-sizing: border-box;` in several style properties. 
What about assigning this property for `body` selector?

Reply:
Ok. I'll correct it =)